### ISSUE TITLE
NullReferenceException while trying to set up an AtomPub provider

### ISSUE THREAD

**Issue by tugberkugurlu** [OpenLiveWriter/OpenLiveWriter#40]
Getting this for [this blog](https://github.com/tugberkugurlu/BLoggy) which has AtomPub integration works fine with WLW:

![image](https://cloud.githubusercontent.com/assets/328122/11697780/3a94eb48-9eb3-11e5-9479-a2059044fa08.png)

**Comment by tugberkugurlu** [comment]
Tried to see what was going on though Fiddler and it seems like it's failing right after one of the images inside the blog provider gives 404:

```
HTTP/1.1 404 Not Found
Server: Apache
Content-Length: 1092
Content-Type: text/html; charset=ISO-8859-1
Date: Wed, 09 Dec 2015 20:30:38 GMT
Connection: keep-alive
Set-Cookie: SONYCOOKIE1=2892277952.20480.0000; path=/
```

I am not sure why it is trying to grab the image though.

**Comment by timheuer** [comment]
@tugberkugurlu is there a hosted site that we can set up a Bloggy test account?

**Comment by gduncan411** [comment]
1: Some of the related issues are noted/commented on in #39 
2: For AtomPub, Microsoft's Channel 9 uses it. That's where I've been doing my AtomPub testing... If you need help there let me know and I'll help get you setup

**Comment by timheuer** [comment]
Dupe of #39 which should be fixed with PR #87

**Comment by tugberkugurlu** [comment]
@timheuer unfortunately not :s only way is to clone the repo and run it locally which is a bit pain :disappointed:
